<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registro</class>
 <widget class="QDialog" name="registro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>481</width>
     <height>612</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>245</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 103, 43);</string>
      </property>
      <property name="text">
       <string>REGISTER</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string extracomment="Name"/>
      </property>
      <property name="placeholderText">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="placeholderText">
       <string>Surname</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="placeholderText">
       <string>DNI</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="placeholderText">
       <string>Phone</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="placeholderText">
       <string>E-mail</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="styleSheet">
       <string notr="true">qproperty-echoMode: 2;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="styleSheet">
       <string notr="true">qproperty-echoMode: 2;
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="placeholderText">
       <string>Confirm Password</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(0, 103, 43);
color: rgb(255, 255, 255);
font: 900 9pt &quot;Arial Black&quot;;</string>
      </property>
      <property name="text">
       <string>Confirm Registration</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
